applications/app/views/fragments/crosswords/crosswordContent.scala.html

```diff
@@ -36,6 +38,11 @@ <h3 class="crossword__clues-header">Down</h3>
                     </div>
                 </div>
             </div>
+            @if(CommercialSwitch.isSwitchedOn && shouldShowAds(crosswordPage)) {
+                <div class="content__secondary-column hide-until-wide" aria-hidden="true">
+                @fragments.commercial.standardAd("right", Seq("mpu-banner-ad"), Map("desktop" -> Seq("300,250", "300,274", "300,600", "fluid")))
```

Comment:
Is 300x274 a valid size?

Reply:
It's the size for the google backfill (ad label funzies)

Reply:
Good question. I didn't make it up though!

https://github.com/guardian/frontend/search?q=300%2C274&unscoped_q=300%2C274

Reply:
Ah, thanks @ScottPainterGNM.